### PR TITLE
clean: using DOCMOSIS_KEY as env variable

### PR DESCRIPTION
**Pull request by akomisarek**
### Change description ###

Update README file and use DOCMOSIS_KEY from env variable

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,15 @@ To enable retry upon test failure please set `TEST_RETRIES` environment variable
 TEST_RETRIES=2 yarn test
 ```
 
+### Docmosis Tornado
```
Comment (geek-soft) [inline README.md]: I am not sure it is the right place. With `H3` it makes it part of `Testing` section which IMO is not related to running tests. I mean not directly.

Comment (geek-soft) [reply README.md]: On second thought whole readme is bit wired because in getting started it focuses on E2E tests.

Comment (akomisarek) [reply README.md]: Yeah to be honest I was not sure where to put it, would use some cleaning.

Comment (akomisarek) [reply README.md]: Did second level header above testing.,

**Pull request merged**